### PR TITLE
Restructured Project + DuckDb Bug Fixed

### PR DESCRIPTION
**Pull request by SaiArrow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,37 +7,33 @@
...
     "vega-transform-db": "^1.0.0",
     "vega-transform-omnisci-core": "^0.0.10",
     "vega-transforms": "^4.9.0"
   },
   "scripts": {
-    "cleanup": "rm -rf .cache dist",
-    "dev": "npm run cleanup && parcel ./index.html",
-    "start": "parcel index.html",
-    "build": "parcel build --public-url '/scalable-vega/' index.html",
+    "build": "yarn --frozen-lockfile && cd ./packages/scalable_vega && yarn --frozen-lockfile",
```
Comment (domoritz) [inline package.json:20]: Use the yarn workspace command and `prebuild`

**Review by domoritz — CHANGES_REQUESTED** [review]

**Review by JunranY — APPROVED** [review]

File: packages/scalable-vega/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "scalable-vega",
```
Comment (domoritz) [inline packages/scalable-vega/package.json:2]: The name is already taken so you need to change it.

File: packages/server/server.ts [hunk 3]
```diff
@@ -171,13 +237,27 @@ app.post('/createSql', async (req: any, res: any) => {
 		console.log('built postgres schema: ' + JSON.stringify(schema));
 		const createTableQueryStr = createTableQueryStrFor(req.body.name, schema);
 		console.log('running create query: ' + createTableQueryStr);
-		if (flag){
+		if (flag==1){
 			await client.query(createTableQueryStr);
 		}
+/*		else if (flag==2){
```
Comment (domoritz) [inline packages/server/server.ts:243]: remove this if it's not used

**Review by domoritz — APPROVED** [review]

File: packages/scalable-vega/package.json (hunk 2 above)
Comment (SaiArrow) [reply packages/scalable-vega/package.json:2]: any suggestions ?

File: packages/server/server.ts (hunk 3 above)
Comment (domoritz) [reply packages/server/server.ts:243]: ok, add a comment

File: packages/scalable-vega/package.json (hunk 2 above)
Comment (domoritz) [reply packages/scalable-vega/package.json:2]: @leibatt?

**Pull request merged**